### PR TITLE
Fixes runtime in manifold.dm and manifold4w.dm

### PR DESCRIPTION
**Pull request by nicbn**

### CODE REVIEW

File: code/modules/atmospherics/machinery/pipes/manifold.dm [hunk 1]
```diff
@@ -18,9 +18,9 @@
 	var/mutable_appearance/center
 
 /obj/machinery/atmospherics/pipe/manifold/Initialize()
+	. = ..()
 	icon_state = ""
```
Comment (ShizCalev) [inline code/modules/atmospherics/machinery/pipes/manifold.dm]: uh

**Comment by optimumtact** [comment]
this makes no sense

**Comment by nicbn** [comment]
Yeah but it works and as it adds checks to make sure nothing breaks

**Comment by nicbn** [comment]
#41957 fixes it properly

**Pull request closed without merging**